src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,58 @@
+// Libraries
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+// Styles
+import './PostInfo.scss';
+
+// Types
+import { Comment } from '../../types/Comment';
+
+// Components
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+type Props = {
+  title: string;
+  userEmail: string | undefined;
+  userName: string | undefined;
+  body: string;
+  comments: Comment[];
+};
+
+// Component
+export const PostInfo: React.FC<Props> = ({
+  title,
+  userEmail,
+  userName,
+  body,
+  comments,
+}) => (
+  <li className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {title}
+      </h3>
+
+      <UserInfo
+        userName={userName}
+        userEmail={userEmail}
+      />
+    </div>
+
+    <p className="PostInfo__body">
```

Comment:
Consider using a more descriptive classname for the component's elements, like 'post-info__body'.